pkg/api/v1/helpers.go

```diff
@@ -326,6 +328,10 @@ func GetTolerationsFromPodAnnotations(annotations map[string]string) ([]Tolerati
 	return tolerations, nil
 }
 
+func GetPodTolerations(pod *Pod) ([]Toleration, error) {
```

Comment:
It's in the mid term of api-machinery change that the PR initially sent, now this duplication no longer exists.